error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/BugCheckerRules.java

```diff
@@ -25,6 +26,7 @@ BugCheckerRefactoringTestHelper before(BugCheckerRefactoringTestHelper helper) {
     }
 
     @AfterTemplate
+    @CanIgnoreReturnValue
     BugCheckerRefactoringTestHelper after(BugCheckerRefactoringTestHelper helper) {
       return helper;
     }
```

Comment:
It might be nice to open a PR upstream to skip suggesting `@CanIgnoreReturnValue` on `@AfterTemplate`. After all, such methods are not really meant to be invoked.

Reply:
I'm totally up for that, will create it tomorrow morning 🦺.

Reply:
Consider first checking the internal upgrade PR; as I mention there, there are a bunch of other (non-Refaster) cases in which adding `@CanIgnoreReturnValue` also doesn't really make sense.